### PR TITLE
Fix(web): `Dropdown` can be placed in non-scrollable `Modal` #DS-939

### PR DESCRIPTION
**Pull request by adamkudrna**
### Additional context

image

### Issue reference

https://jira.almacareer.tech/browse/DS-939

### CODE REVIEW

File: packages/web/src/scss/components/Modal/index.html
Comment (literat) [inline packages/web/src/scss/components/Modal/index.html:1]: I think this change will affect the visual tests. So it should be also in the web-react and web-twig packages as well.

Comment (crishpeen) [reply packages/web/src/scss/components/Modal/index.html:1]: and updated screenshot 😬

File: packages/web/src/scss/components/Modal/index.html [hunk 1]
```diff
@@ -666,6 +666,109 @@ <h2 id="example-non-scrolling-modal-title" class="ModalHeader__title">
...
+      data-spirit-toggle="modal"
+      data-spirit-target="#example-dropdown"
+      aria-controls="example-dropdown"
+      aria-expanded="false"
+    >
+      Open Non-Scrolling Modal with Dropdown
+    </button>
+
+    <!-- Modal: start -->
+    <dialog id="example-dropdown" class="Modal Modal--center" aria-labelledby="example-basic-title">
```
Comment (crishpeen) [inline packages/web/src/scss/components/Modal/index.html:681]:
```suggestion
    <dialog id="example-dropdown" class="Modal Modal--center" aria-labelledby="example-dropdown-title">
```

**Review by pavelklibani — APPROVED** [review]

File: packages/web/src/scss/components/Modal/index.html
Comment (adamkudrna) [reply packages/web/src/scss/components/Modal/index.html:1]: I created an integration branch for it.

**Pull request merged**